src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.SortCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new SortCommand object.
+ */
+public class SortCommandParser implements Parser<SortCommand> {
+    /**
+     * Parses {@code userInput} into a command and returns it.
+     *
+     * @param args
```

Comment:
I think you could explain a little bit on what is `args` and add a fullstop to the statement